src/assets/translations/en/main.json

```diff
@@ -632,12 +632,20 @@
   "trade": {
     "trade": "Trade",
     "allowance": "Allowance",
+    "slippageInfo": "Your transaction will revert if the price changes unfavorably by more than this percentage.",
     "allowanceTooltip": "The amount you will grant the smart contract permission to use on your behalf.",
     "searchingRate": "Fetching rate...",
     "noRateAvailable": "No rate available.",
     "previewTrade": "Preview Trade",
     "rate": "Rate",
     "tradeFrom": "Trade from",
+    "slippage": {
+      "maxSlippage": "Max Slippage",
+      "auto": "Auto",
+      "custom": "Custom",
+      "warning": "Your transaction may be frontrun and result in an unfavorable trade.",
+      "lowSlippage": "Slippage below 0.05% my result in a failed transaction."
```

Comment:
```suggestion
      "lowSlippage": "Slippage below 0.05% may result in a failed transaction."
```

Reply:
With regards to my comment above, maybe this could include "(incurring transaction fees)" or something like this?

It's a common complaint I see in support channels for bridges/DEXes, new users don't expect fees for something that fails.